src/app/page.tsx

```
@@ -10,7 +10,10 @@ import {
   Alert,
   Stack,
   Flex,
+  HStack,
+  Text,
 } from '@chakra-ui/react';
```

ESLint

[error] Unable to resolve path to module '@chakra-ui/react'.

(import-x/no-unresolved)